client/gutenberg/extensions/publicize/connection.jsx

```diff
@@ -12,14 +12,27 @@
  */
 import { Component } from '@wordpress/element';
 import { Disabled, FormToggle, Notice, ExternalLink } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
 import { withSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 import PublicizeServiceIcon from './service-icon';
+import getSiteFragment from 'gutenberg/extensions/presets/jetpack/editor-shared/get-site-fragment';
 
+/**
+ * Return a link to the Sharing page, whether it's on Calypso or WP Admin.
+ *
+ * @returns {string} Link to Sharing page.
+ */
+const getSharingLink = () =>
+	getSiteFragment()
+		? // If running in WP.com wp-admin or in Calypso, we redirect to Calypso sharing settings.
+		  `https://wordpress.com/sharing/${ getSiteFragment() }`
```

Comment:
Totally agree - now that we're introducing a site fragment property in the global variable, it could be the right time to introduce an `environment` one. Could save us a lot of trouble and ugly checks.

Reply:
Publicize section will be removed from Jetpack so I'm removing this.